media-sound/cantata/cantata-2.5.0.recipe

```diff
@@ -69,6 +69,7 @@ BUILD_PREREQUIRES="
 	cmd:cmake
 	cmd:gcc$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
+	cmd:lrelease$secondaryArchSuffix
```

Comment:
```suggestion
	cmd:lrelease$secondaryArchSuffix >= 5
```
Almost all of the recipes specify a version requirement for `lrelease`.  The package built fine without this change but we might as well keep it consistent with the other recipes.

Reply:
Agreed, changed in latest push along with the copyright.